client/signup/steps/jpo-homepage/style.scss

```diff
@@ -40,11 +47,140 @@
 	}
 }
 
-.jpo__homepage-description {
+.jpo-homepage__row .card:first-child {
+	@include breakpoint( ">480px" ) {
+		margin-right: 2%;
+	}
+    
+    @media (max-width: 480px) {
+		margin-right: 10px;
+		width: 100%;
+   		text-align: left;
```

Comment:
Hmm… doesn't have multiple tabs in my text editor. Redid anyways to see if that fixes how it displays here.